web/app/kela/KelaVersiohistoria.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'baret'
+import Atom from 'bacon.atom'
+import Http from '../util/http'
+import {ISO2FinnishDateTime} from '../date/date'
+import Text from '../i18n/Text'
+import Link from '../components/Link'
+
+export const KelaVersiohistoria = ({opiskeluoikeus, oppijaOid}) => {
+  const opiskeluoikeudenOid = opiskeluoikeus.oid
+  const versionumero = opiskeluoikeus.versionumero
+  if (!(opiskeluoikeudenOid || versionumero)) return null
+
+  const historyObs = Http.cachedGet(`/koski/api/luovutuspalvelu/kela/versiohistoria/${opiskeluoikeudenOid}`)
+  const showHistoryAtom = Atom(false)
+
+  return (
+    <div className='versiohistoria'>
+      {historyObs.combine(showHistoryAtom, (history, showHistory) => {
+        return (<>
```

Comment:
Muistelisin, että @mkataja joskus puhui, ettei tämä Reactin uudempi fragment-syntaksi `<>` toimisi oikein `baret`in kanssa. Onko se sittemmin korjattu, vai eikö tässä vain törmää sen ongelmiin?